pkg/ddl/backfilling_dist_scheduler.go

```diff
@@ -451,6 +461,8 @@ func splitSubtaskMetaForOneKVMetaGroup(
 			return nil, errors.Errorf("invalid range, startKey: %s, endKey: %s",
 				hex.EncodeToString(startKey), hex.EncodeToString(endKey))
 		}
+		ts := oracle.ComposeTS(physical, logical)
+		logical++
```

Comment:
does this breaks contraction that ts be different among "transaction"(we're not using transaction though)

Reply:
This is needed by a featutre request, I'll open a issue to explain it.